MIP/MIP-56/README.md

```diff
@@ -0,0 +1,68 @@
+# MIP-56: Rate-Limiter for the Native Bridge
+- **Description**: A rate limitation mechanism for the native bridge.
+- **Authors**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Abstract
+
+The **Rate-Limiter** for the Native Bridge is introduced to de-escalate the risk of double-spends through the bridge by limiting the number of tokens that can be transferred during a certain time frame. The rate limit is determined by the reaction time of the bridge operator, called the **Risk Period**, and the value locked in the **Security Fund**, see [MIP-50](https://github.com/movementlabsxyz/MIP/pull/50).
+
+The Native Bridge Rate Limiter is implemented through a contract on L1. This contract is governed by the Aptos governance framework, see [MIP-48](https://github.com/movementlabsxyz/MIP/pull/48/), and can be updated through a governance process.
+
```

Comment:
If the governance is on L2 the rate limit and other governance related function should be implemented on L2?

Reply:
is the governance on L2 due to operational (gas) costs?

Reply:
Good question. I think we just initially thought of  governance on the L2 because we are aptos first. But perhaps governance on the L1 makes more sense for us. To align those decisions and execute on the L1.

Reply:
i think at least initially we should decide on a particular layer, which would require that most things happen on that layer. which would mean that Rate-Limiter, Bridge-Security-Fund, Governance would need to be on the same layer. 

Considerations: 
- Choice for L1 : Ethereum - alignment
- Choice for L2 : Cost and Latency

Reply:
Surely L1. We don't care about cost for L1, as governance transactions are infrequent. High Gas could be considered a good thing for governance, i.e: its expensive to submit a proposal, do it thoughtfully.

Reply:
Also we can just use already existing protocols such as snapshot https://snapshot.org/#/ 

Or the MolochDAO contracts (I wouldn't recommend), snapshot better.

Reply:
For the governance, I think it's because Aptos framework implements one that we wanted to use.

Reply:
> perhaps, for now any arguments passed to it are set via restricted functions

so who sets these values if not governance?

Reply:
a trusted signer, us. For Bilbao Model, as we wont have any governance implemented for initial mainnet. Only post.